python/tests/test_preequilibration.py

```diff
@@ -243,26 +243,36 @@ def test_raise_postequilibration_with_adjoints(preeq_fixture):
     model, solver, edata, edata_preeq, \
         edata_presim, edata_sim, pscales, plists = preeq_fixture
 
+    # this needs to fail unless we remove presimulation
+    solver.setSensitivityMethod(amici.SensitivityMethod.adjoint)
+
+    rdata = amici.runAmiciSimulation(model, solver, edata)
+    assert rdata['status'] == amici.AMICI_ERROR
+
+    edata.t_presim = 0.0
+    edata.fixedParametersPresimulation = ()
+
     rdatas = {}
-    for sensi_meth in [amici.SensitivityMethod_forward,
-                       amici.SensitivityMethod_adjoint]:
+    for sensi_meth in [amici.SensitivityMethod.forward,
+                       amici.SensitivityMethod.adjoint]:
         # set sensi method
         solver.setSensitivityMethod(sensi_meth)
         solver.setNewtonMaxSteps(0)
         solver.SteadyStateSensitivityMethod = \
-            amici.SteadyStateSensitivityMode_simulationFSA
+            amici.SteadyStateSensitivityMode.simulationFSA
         # add rdatas
         rdatas[sensi_meth] = amici.runAmiciSimulation(model, solver, edata)
+        assert rdatas[sensi_meth]['status'] == amici.AMICI_SUCCESS
 
     for variable in ['llh', 'sllh']:
```

Comment:
I didn't add it in my commits, but checked it rather by hand:
In an ideal world, also sx0 and sx_ss are checked for here...